CHANGELOG.md

```diff
@@ -33,6 +33,162 @@ All sections are explained on the link above, they are all optional, and each of
 
 <!-- New entries go after this line -->
 
+## v2.5.0 - 2017-10-@@@@@@
+
+For more details, [see the full changelog](https://github.com/thelounge/lounge/compare/v2.4.0...v2.5.0) and [milestone](https://github.com/thelounge/lounge/milestone/22?closed=1).
+
+Our [v2.3.0 release](https://github.com/thelounge/lounge/releases/tag/v2.3.0) was big, well, v2.5.0 is even bigger! 🎉
+
+If you are a client user, you will first notice that The Lounge is now using your OS system font, then that status messages (joins, parts, nick changes, etc.) are now condensed with each other.
```

Comment:
`If you are a client user, you will notice that The Lounge is now using your operating system's font, and that status messages (joins, parts, nick changes, etc.) are now condensed with each other.`